pymc3/step_methods/hmc/sgmcmc.py

```diff
@@ -56,61 +54,51 @@ class BaseStochasticGradient(object):
         Optional model for sampling step. Defaults to None (taken from context)
     """
 
-    def __init__(self,
-                 vars=None,
-                 batch_size=128,
-                 n_iter=100,
-                 total_size=None,
-                 step_size=0.25,
-                 burn_in=100,
-                 minibatch_tensors=None,
-                 minibatches=None,
-                 model=None,
-                 **kwargs):
-
-        model = pm.modelcontext(model)
+    def __init__(self, vars=None, batch_size=128, n_iter=100, total_size=None, 
+            step_size=0.25, burn_in=100, minibatch_tensors=None, minibatches=None,
+            model=None, **kwargs):
+        
+        model = modelcontext(model)
 
         if vars is None:
             vars = model.cont_vars
-        vars = pm.inputvars(vars)
+        vars = inputvars(vars)
 
         self.batch_size = batch_size
         self.n_iter = n_iter
         self.total_size = total_size
-        _value_error(total_size != None,
-                     'Total Size of training data has to be specified')
+        _value_error(total_size!=None, 'Total Size of training data has to be specified')
         self.step_size = step_size
         self.burn_in = burn_in
-
+        
```

Comment:
these shouldn't turn up, best to install a plugin in your editor that automatically deletes these.